File: day14.java


	// Add your code here
    Difference(int[] elements){
        this.elements = elements;
        this.maximumDifference =0;
    }


    public void computeDifference(){
        for(int i=0; i<this.elements.length-1; i++){
            for(int j=i+1; j< this.elements.length; j++){
                if(Math.abs(this.elements[i] - this.elements[j]) > this.maximumDifference){
                    this.maximumDifference = Math.abs(this.elements[i] - this.elements[j]);
                }
            }
        }
    }

File: day14.py

        self.maximumDifference=0
        self.diffList =[]
    # Add your code here

    def computeDifference(self):
        for i in range(len(self.__elements)-1):
            for j in range(i+1,len(self.__elements)):
                self.diffList.append(abs(self.__elements[i]-self.__elements[j]))
        self.maximumDifference = max(self.diffList)
